docs/source/concepts/calibration/TubeCalibrationCorelli.rst

```diff
@@ -0,0 +1,116 @@
+.. _Tube_Calibration_Corelli:
+
+Introduction
+============
+
+Mantid has specific functionality for the Corelli instrument at Oak Ridge National Laboratory (ORNL). The Python scripts are located at: **MANTIDINSTALL**/scripts/corelli/Calibration (`github link <https://github.com/mantidproject/mantid/tree/master/scripts/corelli/calibration>`__).
```

Comment:
It may be clearer that this describes a filepath if it were surrounded in double-backticks to denote code